src/figuresClasses.test.ts

```diff
@@ -116,7 +116,7 @@ describe('Method "getArea" of class "Circle"', () => {
     const circle = new Circle('red', 6);
 
     expect(circle.getArea())
-      .toBe(113.09);
```

Comment:
Mathematically you are right (113,0973354) - but for the future - please report such cases to the chat first

In real projects changing the tests assets can happen in different ways, and often just re-writing them is not what you should do